### PR TITLE
feat(i18n): Translate Events view page labels

### PR DESCRIPTION
**Pull request by bdauria**
## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 2.8.x
- [ ] 18.10.x
- [ ] 19.04.x
- [ ] 19.10.x
- [x] 20.04.x (master)

### CODE REVIEW

File: lang/es_ES.UTF-8/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -14295,3 +14295,192 @@ msgstr "Use SSL/TLS"
 
 msgid "Allow self signed certificate"
 msgstr "Permitir certificado autofirmado"
+
+msgid "Acknwoledged by"
```
Comment (kduret) [inline lang/es_ES.UTF-8/LC_MESSAGES/messages.po:14299]:
```suggestion
msgid "Acknowledged by"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -14842,3 +14842,192 @@ msgstr "Eléments verrouillés"
 
 msgid "This value needs to be an integer lesser than"
 msgstr "Cette valeur doit être un entier inférieur à"
+
+msgid "Acknwoledged by"
```
Comment (kduret) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:14846]:
```suggestion
msgid "Acknowledged by"
```

File: lang/pt_BR.UTF-8/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -14448,3 +14448,192 @@ msgstr "Use SSL/TLS"
 
 msgid "Allow self signed certificate"
 msgstr "Permitir certificado autoassinado"
+
+msgid "Acknwoledged by"
```
Comment (kduret) [inline lang/pt_BR.UTF-8/LC_MESSAGES/messages.po:14452]:
```suggestion
msgid "Acknowledged by"
```

File: lang/pt_PT.UTF-8/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -14449,3 +14449,192 @@ msgstr "Use SSL/TLS"
 
 msgid "Allow self signed certificate"
 msgstr "Permitir certificado autoassinado"
+
+msgid "Acknwoledged by"
```
Comment (kduret) [inline lang/pt_PT.UTF-8/LC_MESSAGES/messages.po:14453]:
```suggestion
msgid "Acknowledged by"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 5]
```diff
@@ -11808,7 +11808,7 @@ msgstr "PID"
 
 #: centreon-web/www/include/configuration/configServers/listServers.php:117
 msgid "Start time"
-msgstr "Heure de démarrage du programme"
+msgstr "Temps de début"
```
Comment (kduret) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:11811]:
```suggestion
msgstr "Heure de début"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 6]
```diff
@@ -333,7 +333,7 @@ msgstr "graphique de l'hôte"
 
 #: centreon-web/www/class/centreonGraph.class.php:1190
 msgid "Graph"
-msgstr "Graph"
+msgstr "Graphe"
```
Comment (kduret) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:336]:
```suggestion
msgstr "Graphique"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 7]
```diff
@@ -14842,3 +14842,192 @@ msgstr "Eléments verrouillés"
...
+msgstr "Effacer tout"
+
+msgid "Criterias"
+msgstr "Critères"
+
+msgid "Copy"
+msgstr "Copier"
+
+msgid "Command copied to clipboard"
+msgstr "Commande copiée dans le press-papier"
```
Comment (kduret) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:14865]:
```suggestion
msgstr "Commande copiée dans le presse-papier"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 8]
```diff
@@ -14842,3 +14842,192 @@ msgstr "Eléments verrouillés"
...
+msgstr "Durée de l'état actuel"
+
+msgid "Downtime duration"
+msgstr "Durée du temps d'arrêt"
+
+msgid "End date must be greater than start date"
+msgstr "La date de fin doit être supérieure à la date de début"
+
+msgid "End time"
+msgstr "Temps de fin"
```
Comment (kduret) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:14880]:
```suggestion
msgstr "Heure de fin"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 9]
```diff
@@ -14842,3 +14842,192 @@ msgstr "Eléments verrouillés"
...
+msgstr "Groupe d'hôtes"
+
+msgid "Downtime"
+msgstr "Temps d'arrêt"
+
+msgid "In downtime"
+msgstr "En temps arrêt"
+
+msgid "Last 24 hours"
+msgstr "Dernières 24 heures"
```
Comment (kduret) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:14910]:
```suggestion
msgstr "24 dernières heures"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 10]
```diff
@@ -14842,3 +14842,192 @@ msgstr "Eléments verrouillés"
...
+
+msgid "Last 7 days"
+msgstr "Derniers 7 jours"
```
Comment (kduret) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:14913]:
```suggestion
msgstr "7 derniers jours"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 11]
```diff
@@ -14842,3 +14842,192 @@ msgstr "Eléments verrouillés"
...
+
+msgid "Last 31 days"
+msgstr "Derniers 31 jours"
```
Comment (kduret) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:14916]:
```suggestion
msgstr "31 derniers jours"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 12]
```diff
@@ -14842,3 +14842,192 @@ msgstr "Eléments verrouillés"
...
+msgstr "Temps d'arrêt programmé par"
+
+msgid "Set downtime on services attached to host"
+msgstr "Programmer le temps d'arrêt sur les services attachés à l'hôte"
+
+msgid "Last notification"
+msgstr "Dernière notification"
+
+msgid "Last state change"
+msgstr "Changement du dernier état"
```
Comment (kduret) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:14934]:
```suggestion
msgstr "Dernier changement d'état"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 13]
```diff
@@ -14842,3 +14842,192 @@ msgstr "Eléments verrouillés"
...
+
+msgid "Next check"
+msgstr "Prochain contrôle"
+
+msgid "If checked, a notification is sent to the contacts linked to the object to warn that the incident of the resource has been acknowledged"
+msgstr "Si cette option est cochée, une notification sera envoyée aux contacts lié à l'object afin d'avertir que l'incident de la ressource a été acquitté"
```
Comment (kduret) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:14940]:
```suggestion
msgstr "Si cette option est cochée, une notification sera envoyée aux contacts liés à l'objet afin d'avertir que l'incident de la ressource a été acquitté"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 14]
```diff
@@ -14842,3 +14842,192 @@ msgstr "Eléments verrouillés"
...
+msgstr "Pourcentage de changement d'état"
+
+msgid "Resource name"
+msgstr "Nom de la ressource"
+
+msgid "Resource problems"
+msgstr "Problèmes de ressources"
+
+msgid "Rows per page"
+msgstr "Ressources par page"
```
Comment (kduret) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:14955]:
```suggestion
msgstr "Lignes par page"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 15]
```diff
@@ -14842,3 +14842,192 @@ msgstr "Eléments verrouillés"
...
+msgstr "Désactiver l'actualisation automatique"
+
+msgid "Enable autorefresh"
+msgstr "Activer l'actualisation automatique"
+
+msgid "Search help"
+msgstr "Aide pour la recherche"
+
+msgid "This search bar will search matching occurences you type, contained in: host name or alias or address or service description"
+msgstr "Cette barre de recherche va rechercher les ressources correspondant aux occurances que vous tapez, contenus dans: le nom de l'hôte, l'alias, l'adresse, ou bien la description du service"
```
Comment (kduret) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:14973]:
```suggestion
msgstr "Cette barre de recherche va rechercher les ressources correspondantes aux occurences que vous tapez, contenues dans: le nom de l'hôte, l'alias, l'adresse, ou bien la description du service"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 16]
```diff
@@ -14842,3 +14842,192 @@ msgstr "Eléments verrouillés"
...
+
+msgid "You can use regular expression syntax to retrieve more relevant results"
+msgstr "Vous pouvez utiliser des expressions régulières afin afficher des résultats plus pertinants à la recherches"
```
Comment (kduret) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:14976]:
```suggestion
msgstr "Vous pouvez utiliser des expressions régulières afin afficher des résultats plus pertinents à la recherche"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 17]
```diff
@@ -14842,3 +14842,192 @@ msgstr "Eléments verrouillés"
...
+
+msgid "It's possible to specify the criteria you precisely want to search, by using the following syntax:"
+msgstr "Il est possible de spécifier des critères que vous voulez rechercher de manière précise,  en utilisant la syntaxe suivante:"
+
+msgid "Some examples:"
+msgstr "Quelques examples:"
```
Comment (kduret) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:14982]:
```suggestion
msgstr "Quelques exemples:"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 18]
```diff
@@ -14842,3 +14842,192 @@ msgstr "Eléments verrouillés"
...
+msgstr "Commande de temps d'arrêt envoyée"
+
+msgid "Check command sent"
+msgstr "Commande de contrôle envoyée"
+
+msgid "Type of resource"
+msgstr "Type de ressource"
+
+msgid "Unhandled problems"
+msgstr "Problèmes non traîtés"
```
Comment (kduret) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15018]:
```suggestion
msgstr "Problèmes non traités"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 19]
```diff
@@ -14842,3 +14842,192 @@ msgstr "Eléments verrouillés"
...
+
+msgid "Unhandled"
+msgstr "Non traîté"
```
Comment (kduret) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15021]:
```suggestion
msgstr "Non traité"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 20]
```diff
@@ -14842,3 +14842,192 @@ msgstr "Eléments verrouillés"
...
+
+msgid "Oops, something went wrong"
+msgstr "Oups, quelque chose a cloché"
+
+msgid "Seems that your Apache configuration is not up-to-date"
+msgstr "Il semble que votre configuration Apache n'est pas à jour"
```
Comment (kduret) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15027]:
```suggestion
msgstr "Il semble que votre configuration Apache se soit pas à jour"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 21]
```diff
@@ -14842,3 +14842,192 @@ msgstr "Eléments verrouillés"
...
+
+msgid "Contact your Centreon administrator or update the following site"
```
Comment (kduret) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15029]:
```suggestion
msgid "Contact your Centreon administrator or update the following file"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 22]
```diff
@@ -14842,3 +14842,192 @@ msgstr "Eléments verrouillés"
...
+msgstr "Contactez votre administrateur Centreon ou mettez à jour le site suivant"
```
Comment (kduret) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15030]:
```suggestion
msgstr "Contactez votre administrateur Centreon ou mettez à jour le fichier suivant"
```

File: lang/pt_BR.UTF-8/LC_MESSAGES/messages.po [hunk 23]
```diff
@@ -14448,3 +14448,192 @@ msgstr "Use SSL/TLS"
...
+msgid "Unhandled"
+msgstr ""
+
+msgid "Oops, something went wrong"
+msgstr ""
+
+msgid "Seems that your Apache configuration is not up-to-date"
+msgstr ""
+
+msgid "Contact your Centreon administrator or update the following site"
```
Comment (kduret) [inline lang/pt_BR.UTF-8/LC_MESSAGES/messages.po:14635]:
```suggestion
msgid "Contact your Centreon administrator or update the following file"
```

File: lang/pt_PT.UTF-8/LC_MESSAGES/messages.po [hunk 24]
```diff
@@ -14449,3 +14449,192 @@ msgstr "Use SSL/TLS"
...
+msgid "Unhandled"
+msgstr ""
+
+msgid "Oops, something went wrong"
+msgstr ""
+
+msgid "Seems that your Apache configuration is not up-to-date"
+msgstr ""
+
+msgid "Contact your Centreon administrator or update the following site"
```
Comment (kduret) [inline lang/pt_PT.UTF-8/LC_MESSAGES/messages.po:14636]:
```suggestion
msgid "Contact your Centreon administrator or update the following file"
```

File: lang/es_ES.UTF-8/LC_MESSAGES/messages.po [hunk 25]
```diff
@@ -14295,3 +14295,192 @@ msgstr "Use SSL/TLS"
...
+msgid "Unhandled"
+msgstr ""
+
+msgid "Oops, something went wrong"
+msgstr ""
+
+msgid "Seems that your Apache configuration is not up-to-date"
+msgstr ""
+
+msgid "Contact your Centreon administrator or update the following site"
```
Comment (kduret) [inline lang/es_ES.UTF-8/LC_MESSAGES/messages.po:14482]:
```suggestion
msgid "Contact your Centreon administrator or update the following file"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po (hunk 5 above)
Comment (bdauria) [reply lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:11811]: Then I guess we should update the others to stay consistent?

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 26]
```diff
@@ -14842,3 +14842,192 @@ msgstr "Eléments verrouillés"
...
+msgstr "Type de ressource"
+
+msgid "Unhandled problems"
+msgstr "Problèmes non traités"
+
+msgid "Unhandled"
+msgstr "Non traité"
+
+msgid "Oops, something went wrong"
+msgstr "Oups, quelque chose a cloché"
```
Comment (Moujimouja) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15024]: "s'est mal passé" ?

Comment (bdauria) [reply lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15024]: faudra synchorniser avec BAM & Autodiscovery du coup

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 27]
```diff
@@ -14842,3 +14842,192 @@ msgstr "Eléments verrouillés"
...
+msgstr "Date de début"
+
+msgid "End date"
+msgstr "Date de fin"
+
+msgid "Change start date"
+msgstr "Changer la date de début"
+
+msgid "Change start time"
+msgstr "Changer le temps de début"
```
Comment (cgagnaire) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:14898]:
```suggestion
msgstr "Changer l'heure de début"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 28]
```diff
@@ -14842,3 +14842,192 @@ msgstr "Eléments verrouillés"
...
+msgstr "Ouvrir"
+
+msgid "Percent state change"
+msgstr "Pourcentage de changement d'état"
+
+msgid "Resource name"
+msgstr "Nom de la ressource"
+
+msgid "Resource problems"
+msgstr "Problèmes de ressources"
```
Comment (cgagnaire) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:14952]: Singular in english, plural in french?

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 29]
```diff
@@ -14842,3 +14842,192 @@ msgstr "Eléments verrouillés"
...
+msgstr "Activer l'actualisation automatique"
+
+msgid "Search help"
+msgstr "Aide pour la recherche"
+
+msgid "This search bar will search matching occurences you type, contained in: host name or alias or address or service description"
+msgstr "Cette barre de recherche va rechercher les ressources correspondantes aux occurences que vous tapez, contenues dans: le nom de l'hôte, l'alias, l'adresse, ou bien la description du service"
+
+msgid "You can use regular expression syntax to retrieve more relevant results"
+msgstr "Vous pouvez utiliser des expressions régulières afin afficher des résultats plus pertinents à la recherche"
```
Comment (cgagnaire) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:14976]:
```suggestion
msgstr "Vous pouvez utiliser des expressions régulières afin d'afficher des résultats plus pertinents à la recherche"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 30]
```diff
@@ -14842,3 +14842,192 @@ msgstr "Eléments verrouillés"
...
+
+msgid "It's possible to specify the criteria you precisely want to search, by using the following syntax:"
+msgstr "Il est possible de spécifier des critères que vous voulez rechercher de manière précise,  en utilisant la syntaxe suivante:"
```
Comment (cgagnaire) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:14979]:
```suggestion
msgstr "Il est possible de spécifier les critères que vous voulez rechercher de manière précise, en utilisant la syntaxe suivante:"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po (hunk 28 above)
Comment (bdauria) [reply lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:14952]: Yes, I think that's the rule

**Pull request merged**